Automatic Validation ended with:
> 2024-02-19T12:51:25.1799359Z ##[error] One or more errors occurred. (Addition and deletion only permitted in the same package directory.) Addition and deletion only permitted in the same package directory.

(Automated response - build 763.)

--- a/manifests/k/Kubernetes/kwok/0.4.0/Kubernetes.kwok.installer.yaml
+++ b/manifests/k/Kubernetes/kwok/0.4.0/Kubernetes.kwok.installer.yaml
@@ -1,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.12 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Kubernetes.kwok
+PackageVersion: 0.4.0
+InstallerType: portable
+Commands:
+- kwok
+ReleaseDate: 2023-08-23
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/kubernetes-sigs/kwok/releases/download/v0.4.0/kwok-windows-amd64.exe
+  InstallerSha256: 19276C13F5607F6BB08A666D00E827CEC2C4162713E6D762705E3AE905F6B02B
+- Architecture: arm64
+  InstallerUrl: https://github.com/kubernetes-sigs/kwok/releases/download/v0.4.0/kwok-windows-arm64.exe
+  InstallerSha256: 11DED7B714A68A4A44439C066AF2A08DC2ACFF75743394E699115787C107FFAB
+ManifestType: installer
+ManifestVersion: 1.5.0

--- a/manifests/k/Kubernetes/kwok/0.4.0/Kubernetes.kwok.locale.en-US.yaml
+++ b/manifests/k/Kubernetes/kwok/0.4.0/Kubernetes.kwok.locale.en-US.yaml
@@ -1,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.12 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Kubernetes.kwok
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: Kubernetes
+PublisherUrl: https://kubernetes.io/
+PublisherSupportUrl: https://github.com/kubernetes-sigs/kwok/issues
+# PrivacyUrl: 
+Author: The Kubernetes Authors
+PackageName: Kubernetes WithOut Kubelet
+PackageUrl: https://github.com/kubernetes-sigs/kwok
+License: Apache-2.0 License
+LicenseUrl: https://raw.githubusercontent.com/kubernetes-sigs/kwok/main/LICENSE
+Copyright: Copyright (c) The Kubernetes Authors
+# CopyrightUrl: 
+ShortDescription: Kubernetes WithOut Kubelet - Simulates thousands of Nodes and Clusters.
+# Description: 
+Moniker: kwok
+Tags:
+- docker
+- golang
+- k8s-sig-scheduling
+- kubernetes
+- multi-cluster
+- nerdctl
+- simulator
+# ReleaseNotes: 
+ReleaseNotesUrl: https://github.com/kubernetes-sigs/kwok/releases/tag/v0.4.0
+# PurchaseUrl: 
+# InstallationNotes: 
+# Documentations: 
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0

--- a/manifests/k/Kubernetes/kwok/0.4.0/Kubernetes.kwok.yaml
+++ b/manifests/k/Kubernetes/kwok/0.4.0/Kubernetes.kwok.yaml
@@ -1,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.12 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Kubernetes.kwok
+PackageVersion: 0.4.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0

